main/src/com/google/refine/model/Column.java

```diff
@@ -47,13 +48,25 @@
 import com.google.refine.model.recon.ReconConfig;
 import com.google.refine.util.ParsingUtilities;
 
+import io.frictionlessdata.tableschema.Field;
+import io.frictionlessdata.tableschema.TypeInferrer;
+import io.frictionlessdata.tableschema.exceptions.ConstraintsException;
+import io.frictionlessdata.tableschema.exceptions.InvalidCastException;
+
 public class Column implements Jsonizable {
     final private int       _cellIndex;
     final private String    _originalName;
     private String          _name;
     private ReconConfig     _reconConfig;
     private ReconStats      _reconStats;
     
+    // from data package metadata Field.java:
+    private String type = "";
+    private String format = Field.FIELD_FORMAT_DEFAULT;
+    private String title = "";
```

Comment:
What is the difference between the `title` attribute and the `_name` that we already have?
Also, shouldn't these attributes follow the naming convention of the previous ones (start with `_`)?

Reply:
@wetneb He's just following the spec... Title is also typical of an extra field seen from news organizations as well.
https://frictionlessdata.io/specs/table-schema/

"name": "name of field (e.g. column name)",
      "title": "A nicer human readable label or title for the field",

Reply:
Right, that's the spec. it is kind of weird when I first saw the title for the column. But after 2nd thought, it make sense.

Reply:
oh yeah, okay, thanks!